ovirt/resource_ovirt_vm.go

```diff
@@ -502,6 +545,20 @@ func resourceOvirtVMRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func convertOS(os *ovirtsdk4.OperatingSystem) ([]map[string]interface{}, error) {
+	boot, _ := os.Boot()
```

Comment:
Hi @mathianasj , How about use `os.MustBoot()` and `boot.MustDevices()` here as the `defer` function may catch the `panic` if there has any errors? Thanks.

Reply:
@imjoey I had a chance to change these to mustboot and mustdevices, and reran the tests

Reply:
Ok for me. Thanks.